server/sner/plugin/auror_testssl/agent.py

```diff
@@ -0,0 +1,116 @@
+# This file is part of sner4 project governed by MIT license, see the LICENSE.txt file.
+"""
+sner agent auror_testssl module
+"""
+import ipaddress
+import logging
+import socket
+
+from schema import Schema
+
+from sner.agent.modules import ModuleBase
+from sner.version import __version__ as sner_version
+
+logger = logging.getLogger(__name__)
+
+
+class AgentModule(ModuleBase):  # pragma: cover-ignore-if-not-pytestslow
+    """
+    auror_testssl module implementation
+
+    ## target specification
+    hostname;ip_address;port;explicit
+    """
+
+    CONFIG_SCHEMA = Schema({"module": "auror_testssl"})
+
+    def __init__(self):
+        super().__init__()
+        self.loop = True
+
+    # pylint: disable=duplicate-code
```

Comment:
if the code is same we should use the original/refactored implementation instead of disabling duplicate code detection

Reply:
The same code is in another plugin. If we use that code, it will make this plugin dependent on that plugin. I think we should drop original testssl.sh plugin.

Reply:
we keep testssl plugin for now, it is stable and may be used someday for FLAB usecases where auror_ plugins might not fit well as they are opinionated by auror specific usecase

we can keep duplicate code in this case even if it's not a good practice